### DISCUSSION TITLE
npm i errors

### DISCUSSION THREAD

**Discussion by nkinnacostly** [npm/feedback#388] [General]
> npm ERR! code ECONNRESET
npm ERR! network aborted
npm ERR! network This is a problem related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network 
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'

This My Error

I've tried everything, changed network connectivity, ran npm i -d, npm i --force nothing working, please I need help, it's urgent as i've to deliver these project ASAP

**Comment by MylesBorins** [comment]
Hey @heiscostly . This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. Best of luck!

It looks like your error is related to being behind a proxy or network setup.
